docs-source/content/userguide/experimental/istio.md

```diff
@@ -0,0 +1,228 @@
+  job's pods.
+
+### Exposing applications in Istio-enanbled domains
+
+When a domain is running with the experimental Istio support, you should use the Istio
+gateway to provide external access to applications, instead of using an Ingress 
+controller like Traefik.  Using the Istio gateway will enable you to view the 
+traffic in Kiali and to use distributed tracing all the way from the entry point to 
+the cluster, i.e. the Istio gateway.
+
+To configure external access to your domain, you need to create an Istio `gateway` and
+`virtualservice` as shown in the example below:
+
+```
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: domain1-gateway
+  namespace: domain1
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - hosts:
+        - '*'
+      port:
+        name: http
+        number: 80
+        protocol: HTTP
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: domain1-virtualservice
+  namespace: domain1
+spec:
+  gateways:
+    - domain1-gateway
+  hosts:
+    - '*'
+  http:
+    - match:
+        - uri:
+            prefix: /
+        - port: 8001
+      route:
+        - destination:
+            host: domain1-cluster-cluster-1.domain1.svc.cluster.local
+            port:
+              number: 8001
+```
+
+This example creates a gateway that will accept requests with any host name
+using HTTP on port 80, and a virtual service that will route all of
+those requests to the cluster service for `cluster-1` in `domain1` in
+the namespace `domain1`.
+
+Please refer to the [Istio documentation](https://istio.io/docs/tasks/traffic-management/ingress/)
+for more information about providing ingress using Istio.
+
+### Traffic management
+
+Istio provides traffic management capabilities, including the ability to 
+visualize traffic in Kiali.  You do not need to change your applications to use
+this feature.  The Istio proxy (envoy) sidecar that is injected into your pods
+provides this visibility. The experimental Istio support does enable
+traffic management.  The image below shows an example with traffic
+flowing:
+
+* In from the Istio gateway on the left.
+* To a domain called "bobbys-front-end".
+* To a non-WebLogic application, in this case a Helidon microservice
+  called "bobbys-helidon-stock-application".
+* To a second domain called "bobs-bookstore".
+
+In this example you can see how the traffic flows to the cluster services and 
+then to the individual managed servers.
+
+![Traffic visualization with Kiali](/weblogic-kubernetes-operator/images/kiali.png)
+
+You can learn more about [Istio traffic management](https://istio.io/docs/concepts/traffic-management/)
+in their documentation.
+
+### Distributed tracing
+
+Istio provides distributed tracing capabilities, including the ability to view
+traces in Jaeger.  In order to use distributed tracing though, you will need to 
```

Comment:
may want to add something to effect of "unless you are using Heildon which has OpenTracing 1.3 support where JAX-RS resources are traced implicitly and CDI beans can have tracing enabled via @Traced annotation. See https://medium.com/oracledevs/helidon-brings-microprofile-2-2-support-93d85bb4223e "  - or something more succinct to that point :)

Reply:
i changed it to "WebLogic application" -- will save that helidon stuff for the helidon site :)